### PR TITLE
Add unstable test params for core and extension tests

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Enables marking both ADS Core tests and Extension tests with @UNSTABLE@ for running separate from main test runs. 

Lots of duplication with the index.ts stuff - at some point we should look into consolidating that into our own test running like vscode has that all our extensions use but for now this is fine to get up and running quickly.

### CODE REVIEW

File: scripts/test-extensions.bat [hunk 1]
```diff
@@ -8,23 +8,54 @@ set VSCODEUSERDATADIR=%TMP%\adsuser-%RANDOM%-%TIME:~6,5%
...
+echo **************************
 call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\cms --extensionTestsPath=%~dp0\..\extensions\cms\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
-echo starting dacpac tests
-call .\scripts\code.bat --extensionDevelopmentPath=%~dp0\..\extensions\dacpac --extensionTestsPath=%~dp0\..\extensions\dacpac\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
-echo starting schema compare tests
+
+echo *****************************
+echo *** starting dacpac tests ***
+echo *****************************
+call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\dacpac --extensionTestsPath=%~dp0\..\extensions\dacpac\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
```
Comment (Charles-Gagnon) [inline scripts/test-extensions.bat]: oops - forgot to undo this change. But @aaomidi is there a reason you don't have this set for dacpac tests? They seem to run fine with it

Comment (aaomidi) [reply scripts/test-extensions.bat]: You mean nogpu? No reason I may have just forgot.

Comment (Charles-Gagnon) [reply scripts/test-extensions.bat]: Alright, I'll leave it in and see if it breaks anything =)
